server/src/controllers/PayController.js

```diff
@@ -0,0 +1,52 @@
+const PayServices = require('../services/PayServices');
+const stripeSecretKey = process.env.STRIPE_SECRET_KEY || require('../config/secretKeys').stripeSecretKey;
+
+const stripe = require("stripe")('sk_test_Hrs6SAopgFPF0bZXSN3f6ELN');

```

Comment:
```suggestion
const stripe = require("stripe")(stripeSecretKey);
```
Can you set up stripe to use our secret key (the one in the env variable) instead of this one? I believe this may be one you have for your personal account? Also please make sure that it is the env variable you use, not the plaintext value